move_by_symbols.py

```diff
@@ -15,11 +15,20 @@
 
 symbolCache = {}
 
+# remove symbol cache on edit or on close
 class MoveBySymbolsEventListener(sublime_plugin.EventListener):
     def on_modified_async(self, view):
-        if view.id() in symbolCache:
+        if ST3 and view.id() in symbolCache:
+            del symbolCache[view.id()]
+
+    def on_modifies(self, view):
```

Comment:
This looks suspiciously like a typo. Is it really called `on_modifies` with the `s` in the end?